code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1451,6 +1451,29 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	armor_block = min(90,armor_block) //cap damage reduction at 90%
 	var/Iforce = I.force //to avoid runtimes on the forcesay checks at the bottom. Some items might delete themselves if you drop them. (stunning yourself, ninja swords)
 
+	if(H.mind && H.mind.martial_art && istype(H.mind.martial_art, /datum/martial_art/monk))
```

Comment:
this is an awful bit of code and reveals everything that needs to be said about the martial arts system.

It's not self contained, it gets all through other code and snowflakes it up.

This is really bad